### PR TITLE
ui / sidebar: "alerts on top" toggle (tests, styling tk) [ch10933]

### PR DESCRIPTION
**Pull request by maiamcc**
Hello @hyu, @landism,

Please review the following commits I made in branch maiamcc/alerts-on-top:

04941bd1ea3121cfbbe44178264227242043b1d6 (2021-01-29 14:52:59 -0500)
storybook entry for tests with errors

f92bff4822d87502ec87c536fc494bcc65dc6013 (2021-01-29 13:58:16 -0500)
behavior implemented (needs tests)

Code review reminders, by giving a LGTM you attest that:

* Commits are adequately tested
* Code is easy to understand and conforms to style guides
* Incomplete code is marked with TODOs
* Code is suitably instrumented with logging and metrics

### CODE REVIEW

**Comment by hyu** [comment]
I added styling, and fixed the Filter design while I was at it:

File: web/src/OverviewResourceSidebar.stories.tsx [hunk 1]
```diff
@@ -53,3 +54,21 @@ export function TwoResourcesTwoTests() {
...
+export function TestsWithErrors() {
+  let all: Resource[] = [tiltfileResource()]
+  for (let i = 0; i < 8; i++) {
+    let test = oneResourceTest()
+    test.name = "test_" + i
+    if (i % 2 === 0) {
+      if (!test.buildHistory) {
+        throw "unexpectedly empty buildHistory on test resource"
+      }
+      test.buildHistory[0].error = "egads!"
```
Comment (landism) [inline web/src/OverviewResourceSidebar.stories.tsx:67]: TIL you can cut the if statement and use (the non-null assertion operator)[https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html#non-null-assertion-operator], i.e., `test.buildHistory![0].error = "egads!"`

**Review by landism — COMMENTED** [review]
This looks good. Does tests tk mean in a future commit on this pr or a future pr?

**Comment by hyu** [comment]
@landism — I think it means in a future PR.

**Review by landism — APPROVED** [review]

File: web/src/OverviewResourceSidebar.stories.tsx (hunk 1 above)
Comment (hyu) [reply web/src/OverviewResourceSidebar.stories.tsx]: Nice tip, thanks for helping me grok this!

**Pull request merged**